apps/events/models.py

```diff
@@ -761,7 +720,7 @@ def is_attendee(self, user):
         return self.attendees.filter(user=user)
 
     def is_on_waitlist(self, user):
```

Comment:
If you're in here refactoring, could this be renamed to `attendees_on_waitlist`? This method sounds like one you can use to check if a user is on the waitlist, like `is_on_waitlist(sklirg) : bool`.

Reply:
But that's exactly what it does? It just does the filtering in Python because we can't use filter on spliced querysets.

Reply:
Wow, I must be blind.